synthesizer/process/src/stack/execute.rs

```diff
@@ -421,7 +421,15 @@ impl<N: Network> StackExecute<N> for Stack<N> {
             lap!(timer, "Save the transition");
         }
         // If the circuit is in `CheckDeployment` mode, then save the assignment.
-        else if let CallStack::CheckDeployment(_, _, ref assignments, _) = registers.call_stack() {
+        else if let CallStack::CheckDeployment(_, _, ref assignments, constraint_limit) = registers.call_stack() {
```

Comment:
I thought we agreed we would remove this check because it is already in `verify_vk`

Reply:
The overreport test should still fail when the certificate is verified.